### DISCUSSION TITLE
Retry Support

### DISCUSSION THREAD

**Discussion by annkuoQ** [upptime/upptime#184] [Ideas] [answered]
Can you add this feature in the future?
It will reduce false positives.

Maybe retry every five seconds,
and if it failed three times, return the down status.

**Comment by AnandChowdhary** [comment, marked as answer]
We already do that! 😄 We perform the test three times: if we get a "down" response, we do it once again after waiting for 1 second, and then again after 10 seconds: https://github.com/upptime/uptime-monitor/blob/master/src/update.ts#L105.

**Reply by annkuoQ** [reply]
Appreciate your prompt response; it is pretty helpful.

**Comment by wolfier74** [comment]
Hello the ping interval is not regular. Sometimes uptime CI is triggered every 1h or 2h.
